package.json

```diff
@@ -74,6 +74,18 @@
       "import/order": 0
     }
   },
+  "peerDependencies": {
+    "redux-saga": "1.0.1"
+  },
+  "dependencies": {
```

Comment:
@bbortt - I like the peerDependency change, however, I don't think we should move the next/react/redux packages to `dependencies`. I have them in `devDependencies` so that then are not installed when someone installs this package. They are only necessary to run the example app inside of `next-redux-saga` for testing purposes. Thoughts?

Reply:
I thought that is it kind of obvious that a package named `next-redux-saga` brings along `next`, `redux` and `redux-saga`..? But I don't mind moving it back to `devDependencies`.

Reply:
Well ok, actually I'll move it back. Since we don't want multiple packaged version of `react` for sure.